### PR TITLE
Remove min width on image quickview

### PR DESCRIPTION
**Pull request by EJanuszewski**
Only a small change, seems there's no need for a min-width on the container either.

I have tested with various image sizes and all seem to behave as expected, I will send the pull request shortly.

### CODE REVIEW

**Comment by JeffryBooher** [comment]
@EJanuszewski you can tag the issues that a pull request addresses by adding #(issue-number), #(issue-number), etc....  That links the issues to the pull request so that it's easy to follow.

#10180 
#5960

**Comment by EJanuszewski** [comment]
@JeffryBooher Ok thanks, hopefully any newer things I do i'll get it right first time, I'm new to referring pulls and issues onto someone elses project.

**Comment by JeffryBooher** [comment]
@EJanuszewski no worries :)

File: src/styles/brackets.less [hunk 1]
```diff
@@ -404,6 +403,7 @@ a, img {
 
         .image {
             position: relative;
+            height: calc(~'100% - 53px');
```
Comment (peterflynn) [inline src/styles/brackets.less]: Where does the hardcoded value 53px come from?

**Comment by peterflynn** [comment]
Are the brackets.less changes intended to be part of this PR?  The description at top only mentions #10362...

**Comment by EJanuszewski** [comment]
Sorry I should have put a comment in, 53 is the height of the meta data + its margin(38+15).

Fixes for both are just in the brackets.less, it just happened that to fix the chopping off issue also seemed to fix the resizing, hopefully that helps, I'm on my phone so can't type loads.

**Pull request closed without merging**